File: Chapter 03/C#/3.3.1 SRP/After/TacticsEngine.cs
using System;

namespace Chapter03.SRP.After;

// Responsibility 2: Manages only tactical decisions
public class TacticsEngine
{
    public void DetermineBestPosition(Player player) 
    {
        Console.WriteLine($"  [Tactics] Calculating optimal court position for {player.Name}...");
    }
}

File: Chapter 03/Java/src/main/java/com/grokkingsoftwarearchitecture/chapter03/3.3.1 SRP/After/TacticsEngine.java
package com.grokkingsoftwarearchitecture.chapter03.srp.after;

// Responsibility 2: Manages only tactical decisions
public class TacticsEngine {
    public void determineBestPosition(Player player) {
        System.out.println("  [Tactics] Calculating optimal court position for " + player.name + "...");
    }
}